.github/workflows/build.yml

```diff
@@ -38,6 +38,11 @@ jobs:
       - name: Install dependencies
         run: yarn install --check-files
 
+      - name: Build Packages
+        run: yarn turbo run build -- --cache-dir=.turbo-cache
```

Comment:
Any reason not to run `yarn build` to centralize the command logic?

Reply:
Fixing this as a part of my caching PR.